# Calibration accuracy of individual sensors of Sensor Fusion module
int8 UNRELIABLE = 0
int8 LOW = 1
int8 HIGH = 3
int8 MEDIUM = 2

std_msgs/Header header

int8 accelerometer
int8 gyroscope
int8 magnetometer
